From f2574c2f305b1c59de5369872d060a1992491ab8 Mon Sep 17 00:00:00 2001
From: MengLi <meng.li@windriver.com>
Date: Tue, 3 Jul 2018 13:56:12 +0800
Subject: [PATCH 1332/1345] arm64: dts: marvell: add pin control resource
 address for 7040 platform

Add reg property in pinctrl dts node of cp110-master config space.
This commit refer to SDK kernel commit 5ce9b757f3d6("arm64: dts:
marvell: add pinctrl for Armada 8K CP0 and for CP1")

Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/marvell/armada-7040.dtsi |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/marvell/armada-7040.dtsi b/arch/arm64/boot/dts/marvell/armada-7040.dtsi
index 6a8477d..4c9afb7 100644
--- a/arch/arm64/boot/dts/marvell/armada-7040.dtsi
+++ b/arch/arm64/boot/dts/marvell/armada-7040.dtsi
@@ -83,7 +83,8 @@
 	cp110-master {
 		config-space@f2000000 {
 			pinctrl@440000 {
-			compatible = "marvell,a70x0-pinctrl";
+				compatible = "marvell,a70x0-pinctrl";
+				reg = <0x440000 0x20>;
 				cpm_tdm_pins: tdm-pins {
 					marvell,pins = "mpp0", "mpp1", "mpp2",
 					"mpp3", "mpp4", "mpp5";
-- 
1.7.9.5

